pandas/core/frame.py

```diff
@@ -6652,10 +6652,12 @@ def corr(self, method='pearson', min_periods=1):
 
         Parameters
         ----------
-        method : {'pearson', 'kendall', 'spearman'}
+        method : {'pearson', 'kendall', 'spearman', callable}
```

Comment:
I think it'll be `}, or callable on the outside of the options

Reply:
moved it to outside of the options